### PR TITLE
Diagrama sql

### PR DESCRIPTION
**Pull request by marilirulita**
### In this pull request, we interpret a database diagram and implement the SQL needed to build that database.

- [x] Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- [x] Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- [x] Remember to add the FK indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(30) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  PRIMARY KEY(id)
+);
+
```
Comment (SajjadAhmad14) [inline schema_based_on_diagram.sql:7]: Please create indexes on all the foreign keys. It is the basic solution to increase the performance of queries.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi team, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (melaniesigrid) [reply schema_based_on_diagram.sql:7]: Thank you for your advice! I have created indexes on the foreign keys.

**Review by bimbolabuari — APPROVED** [review]
Hi Team 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**